components/Buttons/tests/unit/ButtonsShapeThemerTests.m

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialButtons+ShapeThemer.h"
+#import "MaterialButtons.h"
+#import "MaterialShapeLibrary.h"
+
+@interface ButtonsShapeThemerTests : XCTestCase
+
+@property(nonatomic, strong) MDCButton *button;
+@property(nonatomic, strong) MDCShapeScheme *shapeScheme;
+
+@end
+
+@implementation ButtonsShapeThemerTests
+
+- (void)setUp {
+  self.button = [[MDCButton alloc] init];
+  self.shapeScheme = [[MDCShapeScheme alloc] init];
+}
+
+- (void)tearDown {
+  self.button = nil;
+  self.shapeScheme = nil;
+}
+
+- (void)testMDCButtonShapeThemer {
+  // Given
+  self.shapeScheme.smallSurfaceShape =
+      [[MDCShapeCategory alloc] initCornersWithFamily:MDCShapeCornerFamilyAngled andSize:10];
+  self.shapeScheme.smallSurfaceShape.topRightCorner = [MDCCornerTreatment cornerWithRadius:3.f];
+  self.button.shapeGenerator = [[MDCRectangleShapeGenerator alloc] init];
```

Comment:
Why is this assignment needed? 
Doesn't applyShapeScheme already assigns a MDCRectangleShapeGenerator to the button?